src/search_algorithms/linear_search/GuessTheNumberGame.java

```diff
@@ -0,0 +1,42 @@
+package search_algorithms.linear_search;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class GuessTheNumberGame {
+    public static void main(String[] args) {
+        System.out.println("Input a number between 1 and 100: ");
+        int[] computerArr = new int[100];
+        for (int i = 1; i <= 100; i++) {
```

Comment:
please insert a new line before procedures (for, if, while, etc)

Reply:
Resolved